lib/web_ui/lib/src/engine/recording_canvas.dart

```diff
@@ -356,8 +356,29 @@ class RecordingCanvas {
     _commands.add(PaintDrawShadow(path, color, elevation, transparentOccluder));
   }
 
-  void drawVertices(ui.Vertices vertices, ui.BlendMode blendMode, ui.Paint paint) {
-    throw new UnimplementedError();
+  void drawVertices(ui.Vertices vertices, ui.BlendMode blendMode,
+      ui.Paint paint) {
+    _hasArbitraryPaint = true;
+    _didDraw = true;
+    final Float32List positions = vertices.positions;
+    double minValueX, maxValueX, minValueY, maxValueY;
+    minValueX = maxValueX = positions[0];
+    minValueY = maxValueY = positions[1];
```

Comment:
Is it possible to receive fewer than 2 positions? Maybe `assert`? What does the C++ Flutter do?

Reply:
Done.